tests/wps_restapi/test_processes.py

```diff
@@ -988,8 +991,26 @@ def deploy_process_CWL_direct(self,
         assert proc["outputs"] == [{
             "id": "output",
             "title": "output",
-            "schema": {"type": "string", "contentMediaType": "text/plain"},
-            "formats": [{"default": True, "mediaType": "text/plain"}]
+            "schema": {
+                "oneOf": [
+                    {
+                        "type": "string",
+                        "contentMediaType": "text/plain"
+                    },
+                    {
+                        "type": "string",
+                        "contentMediaType": "text/html"
+                    },
+                    {
+                        "type": "string",
+                        "format": "binary",
+                        "contentMediaType": "application/pdf",
+                        "contentEncoding": "base64"
+                    }
+                ]
+            },
+            "formats": [{"default": True, "mediaType": "text/plain"}, {"mediaType": "text/html"},
+                        {"mediaType": "application/pdf"}]
```

Comment:
Not sure I follow, the assert is related to the outputs for the process description following `self.deploy_process_make_visible_and_fetch_deployed(body, p_id, assert_io=False)` . In `get_cwl_docker_python_version` cwl outputs are already defined using            
```
 "outputs": {
                "output": {
                    "type": "File",
                    "outputBinding": {
                        "glob": "stdout.log"
                    },
                }
            },
```
not sure where specifying the outputs expected for the process would fit there

Reply:
What I mean is that it looks like all tests after use the same `assert` against the process description contents. So, rather than having it copied many times, the `assert` could be moved inside `deploy_process_make_visible_and_fetch_deployed` to have only 1 place to maintain.